README.md

```diff
@@ -77,3 +78,5 @@ docker exec shop-db-1 /bin/mysqldump -padmin presta > dump.sql
 docker exec shop-db-1 /bin/mysqldump -padmin presta ps_table > dump.sql
 ```
 
+### Inicjalizacja produktów
+Należy najpierw odpalić scrapera (scraper/scraper.py, można uruchomić np. w PyCharmie), żeby pobrały się zdjęcia produktów. Następnie należy otworzyć product-init/ProductInit.sln w VisualStudio i uruchomić Initializer.cs
```

Comment:
Dobrze najpierw wspomnieć o zainstalowaniu zależności z pliku requirements.txt, albo najlepiej o stworzenie wirtualnego środowiska w Pythonie i tam zainstalować zależności

Reply:
Napisałem o tym, ale jak się otworzy katalog w IDE, to IDE i tak powinno to automcztycznie zrobić